packages/node-cli/src/commands/ListFilesCommandGenerator.js

```diff
@@ -66,22 +66,32 @@ module.exports = class ListFilesCommandGenerator extends BaseCommandGenerator {
 		});
 	}
 
-	_executeAction(answers) {
-		// quote folder path to preserve spaces
-		answers.folder = `\"${answers.folder}\"`;
-		const executionContext = new SDKExecutionContext({
-			command: this._commandMetadata.sdkCommand,
-			params: answers,
-			includeProjectDefaultAuthId: true
-		});
+	async _executeAction(answers) {
+		try {
+			// quote folder path to preserve spaces
+			answers.folder = `\"${answers.folder}\"`;
+			const executionContext = new SDKExecutionContext({
+				command: this._commandMetadata.sdkCommand,
+				params: answers,
+				includeProjectDefaultAuthId: true
+			});
 
-		return executeWithSpinner({
-			action: this._sdkExecutor.execute(executionContext),
-			message: TranslationService.getMessage(LOADING_FILES),
-		});
+			const operationResult = await executeWithSpinner({
+				action: this._sdkExecutor.execute(executionContext),
+				message: TranslationService.getMessage(LOADING_FILES),
+			});
+
+			const actionResultContext = {
+				operationResult: operationResult
+			};
+			return new ActionResult.Builder().withSuccess(actionResultContext).build();
+		} catch (error) {
+			return new ActionResult.Builder().withError(error).build();
+		}
 	}
 
-	_formatOutput(operationResult) {
+	_formatOutput(actionResult) {
+		const operationResult = actionResult._context.operationResult;
```

Comment:
same as before, I think actionResult needs either a `getContext()` or to rename it to simply `context`

Reply:
Done for all commands now